### PR TITLE
Updates permissions for updating Edly staff roles

### PR DESCRIPTION
**Pull request by rehan99000**
**Description:**
This PR updates permissions so only global staff and Edly panel admin users can update course creator and global course creator.

**JIRA:**
https://edlyio.atlassian.net/browse/EDLY-1515

### CODE REVIEW

File: openedx/features/edly/tests/test_utils.py [hunk 1]
```diff
@@ -196,3 +209,14 @@ def test_set_global_course_creator_status(self):
         set_global_course_creator_status(self.request, self.user, False)
         assert self._get_course_creator_status(self.user) == 'unrequested'
         assert not auth.user_has_role(self.user, GlobalCourseCreatorRole(edx_org))
+
+        self.admin_user.is_staff = False
+        self.admin_user.save()
+        with self.assertRaises(PermissionDenied):
+            set_global_course_creator_status(self.request, self.user, True)
+
+        edly_panel_admin_user_group, __ = Group.objects.get_or_create(name=settings.EDLY_PANEL_ADMIN_USERS_GROUP)
```
Comment (burhandodhy) [inline openedx/features/edly/tests/test_utils.py:218]: Why do you add `__` ?

File: openedx/features/edly/utils.py [hunk 2]
```diff
@@ -188,6 +190,10 @@ def set_global_course_creator_status(request, user, set_global_creator):
     from course_creators.models import CourseCreator
 
     request_user = request.user
+    is_edly_panel_admin_user = request_user.groups.filter(name=settings.EDLY_PANEL_ADMIN_USERS_GROUP).exists()
```
Comment (asadmanzoor93) [inline openedx/features/edly/utils.py:193]: `EDLY_PANEL_ADMIN_USERS_GROUP ` is defined in edly_panel_app.api.v1.constants.. previous it was in settings.. kindly use the one's defined in edly_panel_app.

File: openedx/features/edly/tests/test_utils.py (hunk 1 above)
Comment (rehan99000) [reply openedx/features/edly/tests/test_utils.py:218]: because i want to ignore the value

File: openedx/features/edly/utils.py (hunk 2 above)
Comment (rehan99000) [reply openedx/features/edly/utils.py:193]: This is not Edly panel app. this is features Edly app.

**Pull request merged**